account_invoice_import/security/rule.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  © 2016-2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo noupdate="1">
+    <record id="account_invoice_import_config_rule" model="ir.rule">
+        <field name="name">Import Supplier Invoices Configuration multi-company</field>
+        <field name="model_id" ref="model_account_invoice_import_config" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
```

Comment:
Should be ['|',('company_id', 'in', company_ids),('company_id','=',False)]

Reply:
Thanks